robot_base_frame: base_link

transform_tolerance: 0.5

# Actual robot radius is 25.4cm
robot_radius: 0.28
inflater:
  robot_radius: 0.28
  inflation_radius: 0.75

obstacles:
  observation_sources: base_scan tilting_cloud
  base_scan:
    data_type: LaserScan
    topic: base_scan
    marking: true
    clearing: true
    obstacle_range: 2.5
    raytrace_range: 3.0
    min_obstacle_height: 0.2
    max_obstacle_height: 0.22
  tilting_cloud:
    data_type: PointCloud2
    topic: head_camera/depth_downsample/points_filtered
    min_obstacle_height: 0.2
    max_obstacle_height: 2.0
    marking: true
    clearing: true
  z_voxels: 8
  z_resolution: 0.25
  max_obstacle_height: 2.0
